rust/src/ffi.rs

```diff
@@ -1,34 +1,24 @@
 use bdk::bitcoin::secp256k1::Secp256k1;
```

Comment:
I recommend you run a formatter for your Rust code (maybe for the Dart code as well? I don't know if they are easy to use). You can just use `cargo fmt` and it will clean up a lot/most of the small nits you have (small spacing issues, etc.) over the Rust files.

Reply:
Done